web_timeline/README.rst

```diff
@@ -59,6 +60,8 @@ Example:
                           event_open_popup="true"
                           zoomKey="ctrlKey"
                           colors="#ec7063:user_id == false;#2ecb71:kanban_state=='done';">
```

Comment:
Shouldn't this be now Python syntax?

Reply:
Yes, definitely. I've pushed the correction. But actually it was already pyevaled before...